DEVELOPERS.md

```diff
@@ -470,30 +440,41 @@ together as a [service](./agent/service.py).
 
 ### The RAP Controller
 
-The RAP Controller has two main entrypoints:
+The RAP Controller has one entrypoint:
 
 - [controller.main](./controller/main.py) polls its database for
     active jobs and takes appropriate action. This involves creating RUNJOB tasks for
     new jobs, creating CANCELJOB tasks for jobs which have been cancelled, retrieving
     associated tasks for running jobs and updating their status.
-- [controller.sync](./controller/sync.py) handles all communication between the job-server and the RAP Controller. It polls the job-server for active JobRequests, updates its local Jobs table accordingly, and then posts back the details of all Jobs associated with the active JobRequests it received.
-The bulk of the work here is done by the
-[create_or_update_jobs](./controller/create_or_update_jobs.py) module.
+
+Jobs are created via the [RAP API](#rap-api) `create` endpoint and the [create_or_update_jobs](./controller/create_or_update_jobs.py) module does the bulk of the work.
 
 Only the Controller has access to its database of Jobs and Tasks.
 
-### The RAP Controller API
+### The RAP Controller APIs
 
 This is a very simple [Django application](./controller.webapp/) that allows external
-applications and users (currently just the Agent) to communicate with the Controller.
+applications and users to communicate with the Controller.
 
+#### Controller Task API
 It has two endpoints, and uses the same backend-specific token from job-server to
 authenticate. These endpoints are essentially view wrappers around methods in the
 controller's [tasks api module](./controller/task_api.py):
 
 - `/<backend>/tasks/`: returns all active tasks for <backend>
 - `/<backend>/task/update/`: receives information about a task and updates the controller database
 
+#### RAP API
+
+The RAP API is used by non-agent clients to communicate with the RAP Controller. It has four endpoints which are defined in the [rap api views](./controller/webapp/views/rap_views.py) module. They are authenticated by the tokens received as part of jobserver's request.
+
+- `/backend/status/`: returns information about the status of backends. This endpoint is polled every minute.
+- `/rap/cancel/`: cancels jobs within a specific job request.
+- `/rap/create/`: creates jobs for a job request.
+- `/rap/status/`: returns current status of jobs after the controller database has been updated by the `create_or_update_jobs` module. This endpoint is polled every minute.
```

Comment:
```suggestion
- `/rap/status/`: returns current status of jobs in the controller database for the requested RAP IDs. This endpoint is polled by job-server every minute.
```
(The create_or_update_jobs module is used by the /create/ endpoint; the status endpoint will return new jobs that were created by the create endpoint, but only when job-server asks for them. It'll also return new job status as jobs are updated by the agent, so I think the relevant thing is that it'll return the jobs as they currently are in the controller db, regardless of what updated them)